torch/ao/quantization/fx/_model_report/README.md

```diff
@@ -0,0 +1,119 @@
+ModelReport
+========
+
+## ModelReport Overview
+
+The `ModelReport` class is the primary class the user will be interacting with in the ModelReport workflow.
+There are three primary methods to be familiar with when using the ModelReport class:
+
+- `__init__(self, desired_report_detectors: Set[DetectorBase])` constructor that takes in instances of desired detectors and stores them.
+This is so that we can keep track of where we want to insert observers on a detector by detector basis and also keep track of which detectors to generate reports for.
+- `prepare_detailed_calibration(self, model: GraphModule)` &rarr; `GraphModule` takes in a GraphModule and inserts observers into the locations specified by each detector.
+It then returns the passed in GraphModule with the detectors inserted into both the regular module structure as well as the node structure.
+- `generate_model_report(self, fx_model: GraphModule, **kwargs)` &rarr; `Dict[str, Tuple[str, Dict]]` takes in a GraphModule that has been callibrated and generates, for each detector the ModelReport instance was initialized with:
+  - A string-based report that is easily digestable and actionable explaining the data collected by relavent observers for that detector
+  - A dictionary containing statistics collected by the relavent observers and values calculated by the detector for futher analysis or plotting
+
+
```

Comment:
should you talk about `ModelReporterObserver` and it's relationship with detectors as well